lib/nano.d.ts

```diff
@@ -1292,7 +1292,10 @@ declare namespace nano {
     name?: string,
 
     // Can be "json" or "text". Defaults to json.
-    type?: 'json' | 'text'
+    type?: 'json' | 'text',
+
+    // This field sets whether the created index will be a partitioned or global index.
+    partitioned: boolean
```

Comment:
sorry for being a pain, but the `partitioned` field is optional so I think it requires a `?`

Reply:
true, it's done